lib/pco_api/actions/create.ex

```diff
@@ -0,0 +1,33 @@
+defmodule PcoApi.Actions.Create do
+  defmacro __using__(_opts) do
+    quote do
+      import PcoApi.Actions.Create
+      import PcoApi.Record
+
+      def post(url) when is_binary(url), do: create("", url)
+
+      def create(%PcoApi.Record{attributes: _, type: _} = record, url) when is_binary(url) do
+        # TODO: Error handling for when the record isn't created
```

Comment:
Found a TODO tag in a comment: # TODO: Error handling for when the record isn't created